tools/mid2agb/main.cpp

```diff
@@ -58,6 +61,9 @@ bool g_compressionEnabled = true;
         "            -X  48 clocks/beat (default:24 clocks/beat)\n"
         "            -E  exact gate-time\n"
         "            -N  no compression\n"
+        "            -A  defer loop before VOL instruction"
```

Comment:
Are these extensions ok? I couldn't think of better letters to use and since the code only allows one dash, one letter arguments...